### ISSUE TITLE
Node crashes within docker container

### ISSUE THREAD

**Issue by hnzlmnn** [nray-scanner/nray#7]
When running a node within a docker container (running Alpine) the application crashes due to the following error:
```
An error occured: machineid: machineid: open /etc/machine-id: no such file or directory  module=utils.errorhandling src=CheckError
panic: runtime error: slice bounds out of range
```

As the missing id is marked as [non-cricial](https://github.com/nray-scanner/nray/blob/master/core/messageStuff.go#L205) the program crashes [here](https://github.com/nray-scanner/nray/blob/master/core/messageStuff.go#L208).

To support running without `/etc/machine-id`, maybe generating a random id on error could be a solution.

**Comment by hnzlmnn** [comment]
Btw, if I am setting `allowMultipleNodesPerHost: true` this is still a problem. To fix it a simple solution is to write arbitrary data to `/etc/machine-id` on container startup but that's a really ugly solution.

**Comment by edermi** [comment]
Thanks for reporting this bug, I also encountered this problem in some containers and already had a fix, but somehow forgot about it...

Nevertheless, it should be fixed in https://github.com/nray-scanner/nray/commit/d3d3fc8127224315408010bab6fb2fd8d764539a so you may test if it works now.

Going to merge it to develop the next few days if it works so it should be available in the next release.

**Comment by edermi** [comment]
A fixed version 1.0.1 was released: https://github.com/nray-scanner/nray/releases/tag/v1.0.1
